lua/src/macro_cache.cc

```diff
@@ -278,11 +277,13 @@ void macro_cache::_process_host_group(neb::host_group const& hg) {
 void macro_cache::_process_host_group_member(
        neb::host_group_member const& hgm) {
   logging::debug(logging::medium)
-    << "macro_cache: process host group member --- "
-    << "host id = " << hgm.host_id
-    << "host group id = " << hgm.group_id
-    << "host group name = " << hgm.group_name;
-  _host_group_members.insert(hgm.host_id, hgm);
+    << "lua: processing host group member "
+    << " (group_name: '" << hgm.group_name << "', group_id: " << hgm.group_id
+    << ", host_id: " << hgm.host_id << ")";
+  if (hgm.enabled)
+    _host_group_members.insert(hgm.host_id, hgm);
```

Comment:
You need to check if membership is already existing to prevent double insertion (or change container type).

Reply:
My bad, I forgot the "multi"... :-(